lib/logidze/model.rb

```diff
@@ -50,6 +50,9 @@ def at(ts)
 
       object_at = dup
       object_at.apply_diff(version, log_data.changes_to(version: version))
+      object_at.id = id
```

Comment:
If we consider, that results of `at` invocation couldn't access their association, as a bug, of course, I can extract this particular change